orbitize/radvel_utils/compute_sep.py

```diff
@@ -0,0 +1,83 @@
+import numpy as np
+import pandas as pd
+
+from radvel.basis import Basis
+from orbitize.basis import t0_to_tau
+from orbitize.kepler import calc_orbit
+
+def compute_sep(
+    df, epochs, basis, mtot, mtot_err, plx, plx_err, n_planets=1, pl_num=1
+):
+    """
+    Computes a sky-projected angular separation posterior given a 
+    RadVel-computed DataFrame.
+
+    Args:
+        df (pd.DataFrame): Radvel-computed posterior (in any orbital basis)
+        epochs (np.array of astropy.time.Time): epochs at which to compute 
+            separations
+        basis (str): basis string of input posterior (see 
+            radvel.basis.BASIS_NAMES` for the full list of possibilities). 
+        mtot (float): median of total mass distribution (assumed Gaussian).
+        mtot_err (float): 1sigma error of total mass distribution 
+            (assumed Gaussian).
```

Comment:
I think my only comment is about mtot vs. m0. For planets this doesn't matter, but will we run into issues if we're doing binary fits? We usually only have a gaussian prior on m0, not mtot for a binary system. Should the code be able to convert K and mstar to mtot more generally?

Reply:
Definitely, great point. I'll amend that.